src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core._util import CLIError
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+
+def _validate_plan_arg(namespace):
```

Comment:
I don't think it is necessary to initial the function name with an underline. The module name itself already indicate its a private module.